src/core/mac/channel_mask.hpp

```diff
@@ -62,7 +63,7 @@ namespace Mac {
  * It is a wrapper class around a `uint32_t` bit vector representing a set of channels.
  *
  */
-class ChannelMask
+class ChannelMask : public Equatable<ChannelMask>
```

Comment:
skip `ChannelMask`? 
I think inheriting `Euquatable` makes `ChannelMask` less efficient.

Reply:
I was actually considering this. I checked the code size impact and noticed that it did not matter (which I did not expect to be honest). I think there are efficient `memcmp` implementations which handle small size differently so the complier may be optimizaing it already.

It just helps avoding the definitions (src code reduction), but I agree it may better/safer to not use it for `ChannelMask` and `Time`. Will change it.